config/yunikorn-history-server/local.yml

```diff
@@ -0,0 +1,22 @@
+yunikorn:
+  host: 127.0.0.1
+  port: 30001
+  secure: false
+
+db:
+  host: 127.0.0.1
+  port: 30002
```

Comment:
Where does postgres run in this configuration (local.yml)? Is this in a container, or directly on the Linux/MacOS server? Is this file for "localdev" work, or something else?

Reply:
`make kind-all` creates a Kind cluster which exposes following ports via NodePorts (ports must be in the range 30000-32767):
* 30000 -> yunikorn-core (9080)
* 30001 -> yunikocn-service (9889)
* 30002 -> postgres (5432)
* 30003 -> yhs (9898)

Then Yunikorn and Postgres get installed and the respective Kubernetes Services are configured to be of type NodePort and to match the ports above.

After that, you can access those ports on `localhost` without any need for port-forwarding or additional actions.